application/system_processor/decoder/Cdl.py

```
@@ -1,9 +1,9 @@
 from pathlib import Path
 from clp_ffi_py.ir import ClpIrFileReader
-from CDL_CONSTANTS import LINE_TYPE_DELIMITER, LINE_TYPE
-from CdlLogLine import CdlLogLine
-from Variable import Variable
-from CdlHeader import CdlHeader
+from decoder.CDL_CONSTANTS import LINE_TYPE_DELIMITER, LINE_TYPE
```

Ruff (0.8.2)

`decoder.CDL_CONSTANTS.LINE_TYPE_DELIMITER` imported but unused

Remove unused import: `decoder.CDL_CONSTANTS.LINE_TYPE_DELIMITER`

(F401)